apps/docs/pages/guides/database/column-encryption.mdx

```diff
@@ -9,9 +9,27 @@ export const meta = {
   video: 'https://www.youtube.com/v/J9mTPY8rIXE',
 }
 
-Supabase provides a secure method for encrypting columns using [Vault](/docs/guides/database/vault), our Postgres secrets manager. Vault is a Postgres extension with an integrated UI intended to act as a secure global secrets management for your project.
+Supabase provides a secure method for encrypting data using [Vault](/docs/guides/database/vault), our Postgres secrets manager. Vault is a Postgres extension with an integrated UI intended to act as a secure global secrets management for your project.
 
-Vault enables an advanced feature called Transparent Column Encryption (TCE) which provides a safe way to encrypt your data so that it doesn't leak into logs and backups. It can also provide row-level authenticated encryption.
+In addition to the Vault secret storage table, Supabase also enables an advanced feature called Transparent Column Encryption (TCE) which provides a safe way to encrypt columns in your own tables so that they doesn't leak into logs and backups. It can also provide row-level authenticated encryption.
+
+Column Encryption comes with tradeoffs that need to be considered before using it.
+
+    - Encryption and decryption both take time, inserting and selecting encrypted data takes more time than a "plain" column of data.  Queries that must load a lot of rows with encrypted columns will be slower than those that are not encrypted.
+
+    - Encrypted columns should never be indexed.  This is because the index must store the unencrypted value of a column to be useful, which would defeat the purpose of storing values encrypted.  To look up an encrypted column, you must first look up its row by some other unencrypted column, like a primary key.
```

Comment:
```suggestion
    - Encrypted columns should never be indexed.  This is because the index will store the encrypted value of a column, which would not be useful.  To look up an encrypted column, you must first look up its row by some other unencrypted column, like a primary key.
```